mycore-ocfl/src/main/java/org/mycore/ocfl/MCROCFLXMLMetadataManager.java

```diff
@@ -248,7 +252,7 @@ private VersionInfo buildVersionInfo(String message, Date versionDate, String us
     }
 
     @Override
-    public List<MCROCFLMetadataVersion> listRevisions(MCRObjectID id) throws IOException {
+    public List<MCROCFLMetadataVersion> listRevisions(MCRObjectID id) throws IOException, NotFoundException {
```

Comment:
For consistency it would might be better to catch the NotFoundException like in the other methods:
        } catch (NotFoundException e) {
            throw new MCRUsageException("Object '" + objName + "' could not be found", e);
        }